routes/api.php

```diff
@@ -22,3 +22,6 @@
 Route::get('cart/{userId}', "Api\CartController@index");
 Route::put('cart/{userId}', "Api\CartController@update");
 Route::delete('cart/{userId}', "Api\CartController@destroy");
+Route::get('transactions/{userId}', "Api\TransactionController@index");
+Route::get('transactions/{userId}/{trackingNumber}', "Api\TransactionController@show");
+Route::delete('transactions/{userId}/{trackingNumber}', "Api\TransactionController@destroy");
```

Comment:
lets use /{userId}/transactions/{trackingNumber}. I might also change the cart api routing